resources/tests.mjs

```diff
@@ -66,7 +71,8 @@ Suites.push({
         page.getLocalStorage().getItem("javascript-es5");
     },
     tests: [
-        new BenchmarkTestStep(`Adding${numberOfItemsToAdd}Items`, (page) => {
+        new BenchmarkTestStep(`Adding${getNumberOfItemsToAdd()}Items`, (page) => {
```

Comment:
Maybe I should just stick to using `defaultNumberOfItemsToAdd` here, so that the metric names don't change?

Reply:
I think the name should reflect the complexity level, in an effort to avoid comparing a simple test run with a complex test run

Reply:
Maybe we could remove the number here and handle this during reporting time - update the code that consumes the name after the run when reporting in the UI or JSON as either `TodoMVC-LocalStorage (Complexity 1.5)` -> `AddingItems`, or `TodoMVC-LocalStorage` -> `AddingItems (Complexity 1.5)` when complexity !== 1? One tradeoff there is it would report with complexity even in tests that aren't complexity-aware, but that seems maybe fine given that this is an opt-in thing that you'd only use if you know what you're doing.

@rniwa WDYT?

Reply:
Definitely open to either suggestion.  `getNumberOfItemsToAdd()` would work partially if the complexity param is passed-in via url-params only (but that's a bit brittle).

Reply:
Naively I'd do this:
* add the complexity value somewhere in the json report, but not in all tests. Also display it somewhere in the results interface.
* remove any mention of actual counts in the test titles

But I'm not convinced this is right, because we might want the tests themselves to hold this value so that it's clear in reports that we're not at the default complexity.

Also if we want to change the actual titles, I'd like to see the complexity in the test names more than in the steps.

Reply:
- I can add a `non-standard-params` warning in the detailed results page to surface what's happening
- I'd vouch for dropping mentioning the complexity params (much like other score-influencing params, we don't surface them anywhere else)

Reply:
In the latest commit I've just removed the reference to 100 in the step title.